src/dragonfs.c

```diff
@@ -979,6 +979,22 @@ uint32_t dfs_rom_addr(const char *path)
     return base_ptr+entry->data_ofs;
 }
 
+uint32_t dfs_rom_size(const char *path)
+{
+    //Skip initial slash
+    if(path[0] == '/') {
+        path++;
+    }
+    dfs_lookup_file_t *entry = lookup_file(path);
+    
+    if(!entry)
+    {
+        //File not found
+        return 0;
+    }
+    return base_ptr+entry->data_len;
```

Comment:
I'm pretty sure you shouldn't add `base_ptr` here?

Reply:
I definitely shouldn't! Unless I wanted to have a Libultra-style SEGEMENT_END bit!